readthedocs/search/tasks.py

```diff
@@ -168,11 +194,21 @@ def record_search_query(project_slug, version_slug, query, total_results):
     version_qs = Version.objects.filter(project=project, slug=version_slug)
 
     if not version_qs.exists():
+        log.debug(
+            'Not recording the search query because version does not exist. '
+            'project_slug: %s, version_slug: %s' % (
+                project_slug, version_slug
+            )
+        )
         return
 
     version = version_qs.first()
-    SearchQuery.objects.create(
+
+    # make a new SearchQuery object.
+    obj = SearchQuery.objects.create(
         project=project,
         version=version,
         query=query,
     )
+    obj.created = time
```

Comment:
Shouldn't this be an auto field?

Reply:
Yes... but the actual time is when the query was made.
Celery task are not executed *immediately* so the automated time will be wrong.

Reply:
I'd say that's probably OK to use the auto field for this.

Reply:
I used this data for query also, so I think if we use automated data, results will be wrong.
- https://github.com/readthedocs/readthedocs.org/pull/6088/files#diff-2a0ffe6d23a7ca20d7e60b8e1617836bR161
- https://github.com/readthedocs/readthedocs.org/pull/6088/files#diff-2a0ffe6d23a7ca20d7e60b8e1617836bR168